examples/robot_example.cpp

```diff
@@ -0,0 +1,84 @@
+/*!
+ * @file robot_example.cpp
+ * @author J. Diego Caporale <jdcap@seas.upenn.edu>
+ * @brief Example that uses SimpleRobot(a RobotInterfaceDerived class defined in a seperate header file) separating the 
+ *        controller, state update, and state machine from the lower lever control loop. 
+ * @date 2022-07-17
+ * 
+ * @copyright Copyright (c) 2021 The Trustees of the University of Pennsylvania. All Rights Reserved
+ *            BSD 3-Clause License
+ * 
+ */
+#include <vector>
+#include <iostream>
+#include <memory>
+#include "kodlab_mjbots_sdk/common_header.h"
+#include "ManyMotorLog.hpp"
+#include "ModeInput.hpp"
+#include "kodlab_mjbots_sdk/robot_interface.h"
+#include "kodlab_mjbots_sdk/mjbots_control_loop.h"
+
+#include "examples/simple_robot.h"
+
+class SimpleRobotControlLoop : public kodlab::mjbots::MjbotsControlLoop<ManyMotorLog, ModeInput, SimpleRobot>
+{
+    using MjbotsControlLoop::MjbotsControlLoop;
+
+    void CalcTorques() override
+    {
+        robot_->Update();
```

Comment:
I would add some code here that actually uses the robot to set some torques or atleast a comment expressing that idea as a code pattern.